test/node/NodeMetricsProvider.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
```

Comment:
we shouldn't need this one here if we're ignoring this rule in tests? but we probably shouldn't disable it completely in all tests.. I'm good with it either way.. eslint can be painful in test code.